web/themes/custom/votegov/src/sass/components/double-deck-card.scss

```
@@ -0,0 +1,75 @@
+@use "uswds-core" as *;
+@use "variables" as *;
+@use "mixins" as *;
+
+.vote-double-deck-card {
+  --vote-ddc-bg: #{#F7F9FA};
+  @include vote-block-margin-y;
+  background-color: var(--vote-ddc-bg);
+  border: 1px solid $base-dark;
+  line-height: 155%;
+
+  [data-theme="contrast"] & {
+    --vote-ddc-bg: #{$base-white};
+  }
+
+  &__primary {
+    @include u-padding(4);
+
+    @include at-media('tablet') {
+      @include u-display('flex');
+    }
+  }
+
+  &__icon {
+    @include u-margin-bottom(3);
+    @include u-margin-right(3);
+    text-align: center;
+
+    svg {
+      --vote-ddc-icon-fill: #{$base-dark};
+      fill: var(--vote-ddc-icon-fill);
+
+      [data-theme="contrast"] & {
+        --vote-ddc-icon-fill: #{$bg-high-contrast};
+      }
+    }
+  }
+
+  &__heading {
+    @include u-font('serif', 9);
+  }
+
+  &__body {
+    @include u-padding-y(3);
+    font-size: 16px;
+
+    ul {
+      margin: 0;
+
+      @include at-media('tablet') {
+        column-count: 2;
+        column-gap: 7px;
+        max-height: 450px;
+        overflow-y: auto;
+        column-fill: auto;
+      }
+    }
+  }
+
+  &__body_secondary {
+    --vote-ddc-body-secondary-bg: #{$bg-light-cool};
+    background-color: var(--vote-ddc-body-secondary-bg);
+    @include u-padding(4);
```

looks like there is to much space on the left and right for the bottom text. 


```suggestion
    @include u-padding-x(4);
```

---

now that im looking at the figma the mobile and mobile large look to have 2 different spaces on the sides